site/content/kapp-controller/_index.html

```diff
@@ -57,106 +73,132 @@ <h2>Basic Usage</h2>
-      12:52:12AM: create deployment/simple-app (apps/v1) namespace: default
-      12:52:12AM: ---- waiting on 2 changes [0/2 done] ----
-      12:52:12AM: ok: reconcile service/simple-app (v1) namespace: default
-      12:52:12AM: ongoing: reconcile deployment/simple-app (apps/v1) namespace: default
-      12:52:12AM:  ^ Waiting for generation 2 to be observed
-      12:52:12AM:  L ok: waiting on replicaset/simple-app-69f4586488 (apps/v1) namespace: default
-      12:52:12AM:  L ongoing: waiting on pod/simple-app-69f4586488-9qrx2 (v1) namespace: default
-      12:52:12AM:     ^ Pending: ContainerCreating
-      12:52:12AM: ---- waiting on 1 changes [1/2 done] ----
-      12:52:12AM: ongoing: reconcile deployment/simple-app (apps/v1) namespace: default
-      12:52:12AM:  ^ Waiting for 1 unavailable replicas
-      12:52:12AM:  L ok: waiting on replicaset/simple-app-69f4586488 (apps/v1) namespace: default
-      12:52:12AM:  L ongoing: waiting on pod/simple-app-69f4586488-9qrx2 (v1) namespace: default
-      12:52:12AM:     ^ Pending: ContainerCreating
-      12:52:13AM: ok: reconcile deployment/simple-app (apps/v1) namespace: default
-      12:52:13AM: ---- applying complete [2/2 done] ----
-      12:52:13AM: ---- waiting complete [2/2 done] ----
-      Succeeded
-    updatedAt: "2020-12-15T00:52:13Z"
-  inspect:
-    exitCode: 0
-    stdout: |-
-      Target cluster 'https://10.96.0.1:443'
-      Resources in app 'simple-app-ctrl'
-      Namespace  Name                         Kind        Owner    Conds.  Rs  Ri  Age
-      default    simple-app                   Deployment  kapp     2/2 t   ok  -   3s
-      ^          simple-app                   Endpoints   cluster  -       ok  -   3s
-      ^          simple-app                   Service     kapp     -       ok  -   3s
-      ^          simple-app-69f4586488        ReplicaSet  cluster  -       ok  -   3s
-      ^          simple-app-69f4586488-9qrx2  Pod         cluster  4/4 t   ok  -   3s
-      Rs: Reconcile state
-      Ri: Reconcile information
-      5 resources
-      Succeeded
-    updatedAt: "2020-12-15T00:52:13Z"
-{{< / highlight >}}
-    </div>
+    imgpkgBundle:
+      image: k8slt/corp-com-pkg-repo:1.0.0
+{{< / highlight>}}
+    <p>Apply the PackageRepository to your cluster and view available software packages with kubectl:</p>
+{{< highlight bash "">}}
+$ kubectl apply -f repository.yml
+
+$ kubectl get packages
+NAME                             PACKAGEMETADATA NAME   VERSION      AGE
+simple-app.corp.com.1.0.0        simple-app.corp.com    1.0.0        1m50s
+simple-app.corp.com.2.0.0        simple-app.corp.com    2.0.0        1m50s
+simple-app.corp.com.3.0.0-rc.1   simple-app.corp.com    3.0.0-rc.1   1m50s
+{{< / highlight>}}
+    <p>kapp-controller's PackageInstall CR provides users a way to install Packages on a 
+      Kubernetes cluster.</p>
+{{< highlight yaml "">}}
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  name: pkg-demo
```

Comment:
we should probably throw in `namespace: something` to indicate that it is added to a namespace